src/app/History.ts

```diff
@@ -32,6 +32,7 @@ export enum URLParam {
   OPERATION_NODES = 'operationNodes',
   OVERVIEW_TYPE = 'otype',
   QUANTILES = 'quantiles',
+  RANGE_DURATION = 'rangeDuration',
```

Comment:
It may create some conflicts when navigating from one tab to another as there is a process to set the URL to state and viceversa.

I found some race on that logic, so keeping this separated I guess it helps.